toolkit/scripts/rpmops.sh

```diff
@@ -14,7 +14,21 @@ fi
 
 # Additional macros required to parse spec files.
 DIST_TAG=$(make -s -f $REPO_ROOT/toolkit/Makefile get-dist-tag)
-DEFINES=(-D "with_check 1" -D "dist $DIST_TAG")
+RELEASE_NUM=$(make -s -f $REPO_ROOT/toolkit/Makefile get-release-major)
```

Comment:
I think we may need something similar inside a few more places where we manually define the macros:
- `toolkit/scripts/toolchain/check_manifests.sh`,
- `toolkit/scripts/toolchain/build_official_toolchain_rpms.sh`.

Reply:
Hmm, I realize now that the toolchain call to srpmpacker needs these as well

Reply:
Ok, we need to define these explicitly in the go tools. I noticed we only call `DefaultDefines()` in one place (srpmpacker), and we use the distro tag version everywhere else. I unified the two into a single function and removed the special case handling from srpmpacker.